odl/contrib/pytorch/examples/basic.py

```diff
@@ -0,0 +1,77 @@
+"""Demonstration of basic ODL->pytorch integration functionality."""
+
+import numpy as np
+import torch
+import odl
+from odl.contrib.pytorch import TorchOperator
+
+# --- Forward --- #
+
+# Define ODL operator
+matrix = np.array([[1, 2, 3],
+                   [4, 5, 6]], dtype=float)
+odl_op = odl.MatrixOperator(matrix)
+
+# Wrap as torch operator
+torch_op = TorchOperator(odl_op)
+
+# Define evaluation point and wrap into a variable. Mark as
+# `requires_gradient`, otherwise `backward()` doesn't do anything.
+# This is supported by the ODL wrapper.
+x = torch.ones((3,)).type(torch.DoubleTensor)
```

Comment:
`x = torch.DoubleTensor([1, 1, 1])` should work, no?

Reply:
Of course